Products/DataCollector/zenmodeler.py

```diff
@@ -1047,8 +1051,6 @@ def timeMatches(self):
         Check whether the current time matches a cron-like
         specification, return a straight true or false
         """
-        if self.startat is None:
-            return True
```

Comment:
When `timeMatches` is called, `startat` will never be `None`?

Reply:
yes, we checking this here https://github.com/zenoss/zenoss-prodbin/pull/4458/files#diff-ffa1241dd5b49bebb381f208827c9be117c752e396e1c2f120cdd47cda1a1161R893